web/themes/joinup/prototype/sass/components/_sidebar-menu.sass

```diff
@@ -61,7 +61,7 @@
 // Modifiers
 // --------------------------------------------------
 .sidebar-menu__button--mobile
-  margin-top: -80px
+  margin-top: -130px
```

Comment:
Small note after the daily - please try if -200px will solve it, if not, please check if JS will be possible.